pango-sys/src/manual.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013-2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::GType;
+
+#[no_mangle]
+#[cfg(not(feature = "v1_44"))]
+pub fn pango_attr_iterator_get_type() -> GType {
+    0
```

Comment:
Can you remove this for now? Then we could already merge everything except for pango and reduce the risk of merge conflicts.

I'll take care of Pango separately then.

Reply:
done

Reply:
Thanks :)